src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/service/UpsertUnallocatedCaseService.kt

```diff
@@ -52,7 +52,7 @@ class UpsertUnallocatedCaseService(
               unallocatedCaseEntity.crn,
               activeConvictions
             )
-            val caseType = caseTypeEngine.getCaseType(activeConvictions, unallocatedCaseEntity.convictionId)
+            unallocatedCaseEntity.convictionNumber = conviction.convictionNumber
```

Comment:
needed to move this so the convictionNumber is set on new cases before determining the caseType

Reply:
Maybe you could have just passed in `conviction.convictionNumber` to `caseTypeEngine.getCaseType`, but no biggie